ipa-core/src/ff/curve_points.rs

```diff
@@ -42,6 +42,18 @@ impl Vectorizable<1> for RP25519 {
     type Array = StdArray<Self, 1>;
 }
 
+impl Vectorizable<16> for RP25519 {
+    type Array = StdArray<Self, 16>;
+}
+
+impl Vectorizable<64> for RP25519 {
+    type Array = StdArray<Self, 64>;
+}
+
+impl Vectorizable<256> for RP25519 {
```

Comment:
in the chat you mentioned that 256 is causing stack overflow, should we remove this implementation and add a note here?